src/msw/ole/access.cpp

```diff
@@ -1827,8 +1827,28 @@ IAccessible *wxAccessible::GetIAccessibleStd()
 void wxAccessible::NotifyEvent(int eventType, wxWindow* window, wxAccObject objectType,
                         int objectId)
 {
-    ::NotifyWinEvent((DWORD) eventType, (HWND) window->GetHWND(),
-        (LONG) objectType, (LONG) objectId);
+    const HWND hwnd = (HWND)window->GetHWND();
+
+    // send the notification in idle time to be sure it is sent after the change
+    // was fully done in wx code
+    struct SendNotification
```

Comment:
I think this is not going to compile with g++ in non-C++11 mode, template parameters must have extern linkage in C++98.

IOW this struct should be moved out of the function.